### ISSUE TITLE
perplexity of a sentence

### ISSUE THREAD

**Issue by anonymous297** [minimaxir/gpt-2-simple#139]
I have finetuned the model and want to calculate perplexity of an input sentence.
What function can be used for that.

**Comment by pskl** [comment]
This leaves me perplexed 🤔

**Comment by gpt2ent** [comment]
It would really help if we had a function analogous to [gpt2.sample.sample_sentence](https://github.com/minimaxir/gpt-2-simple/blob/ca6bc61d958fd4c474af9a412ace27279b88dd90/gpt_2_simple/src/sample.py#L41) that outputs raw logits of a pre-generated sentence instead of generating samples.
